api/v1/lib/master/calls/calls.go

```diff
@@ -221,6 +228,32 @@ func DestroyVolumes(a mesos.AgentID, v ...mesos.Resource) *master.Call {
 	}
 }
 
+// GrowLocalVolume grows a persistent volume on an agent's ROOT or PATH disks. The request is forwarded asynchronously
+// to the Mesos agent where the persistent volume is located.
+func GrowLocalVolume(a mesos.AgentID, volume mesos.Resource, addition mesos.Resource) *master.Call {
+	return &master.Call{
+		Type: master.Call_GROW_VOLUME,
+		GrowVolume: &master.Call_GrowVolume{
+			AgentID:  &a,
+			Volume:   volume,
+			Addition: addition,
+		},
+	}
+}
+
+// ShrinkLocalVolume shrinks a persistent volume on an agent's ROOT or PATH disks. The request is forwarded
+// asynchronously to the Mesos agent where the persistent volume is located.
+func ShrinkLocalVolume(a mesos.AgentID, volume mesos.Resource, subtract mesos.Value_Scalar) *master.Call {
```

Comment:
mesos.AgentID is optional in protos, should be a pointer parameter

Reply:
I'll make it a pointer. Then should I just rename it to `GrowVolume`/`ShrinkVolume`? Currently it supports only local volumes but me may extend it to support external volumes in the future.